env/anvil.json

```diff
@@ -5,6 +5,8 @@
         "environment": "testnet",
         "network": "anvil",
         "catapultaNetwork": "anvil",
+        "messageCost": "1000000",
+        "maxBatchSize": "50000000",
```

Comment:
FYI. In https://github.com/centrifuge/protocol-v3/pull/495
- `messageCost` will be removed.
- `maxBatchSize` is renamed to `batchGasLimit`